test/qpy_compat/test_qpy.py

```diff
@@ -525,6 +549,8 @@ def generate_circuits(version_str=None):
         output_circuits["controlled_gates.qpy"] = generate_controlled_gates()
         output_circuits["schedule_blocks.qpy"] = generate_schedule_blocks()
         output_circuits["pulse_gates.qpy"] = generate_calibrated_circuits()
+    if version_parts >= (0, 21, 2):
```

Comment:
I've made this 0.21.2 assuming we might throw this in the backports, but if we're leaving it for 0.22.0, feel free to just modify this line.

Reply:
Yeah, that's fine I probably would have just done `> (0.21.1)` to cover either case but this works fine as I was intending to tag this as stable backport potential